aspnet/security/2fa.rst

```diff
@@ -135,55 +111,62 @@ Let us run the Web site and show the two-factor authentication flow.
-- Clicking on the Remember this browser check box will exempt you from needing to use 2FA to log on with that computer and browser. Enabling 2FA and clicking on the  Remember this browser will provide you with strong 2FA protection from malicious users trying to access your account, as long as they don't have access to your computer. You can do this on any private machine you regularly use. By setting  Remember this browser, you get the added security of 2FA from computers you don't regularly use, and you get the convenience on not having to go through 2FA on your own computers. 
+- Enter the code you get in the SMS message. 
+- Clicking on the **Remember this browser** check box will exempt you from needing to use 2FA to log on when using the same device and browser. Enabling 2FA and clicking on **Remember this browser** will provide you with strong 2FA protection from malicious users trying to access your account, as long as they don't have access to your device. You can do this on any private device you regularly use. By setting  **Remember this browser**, you get the added security of 2FA from devices you don't regularly use, and you get the convenience on not having to go through 2FA on your own devices. 
 
 .. image:: 2fa/_static/login2fa8.png
 
-- Log In.
-
-.. image:: 2fa/_static/login2fa9.png
-
-           
 Account lockout for protecting against brute force attacks
 ----------------------------------------------------------
-We recommend you use account lockout with 2FA.Once a user logs in (through local account or social account), each failed attempt at 2FA is stored, and if the maximum attempts (default is 5) is reached, the user is locked out for five minutes (you can set the lock out time with DefaultAccountLockoutTimeSpan).
-The following configures Account to be locked out for 10 min after 10 failed attempts.
+We recommend you use account lockout with 2FA.Once a user logs in (through a local account or social account), each failed attempt at 2FA is stored, and if the maximum attempts (default is 5) is reached, the user is locked out for five minutes (you can set the lock out time with ``DefaultAccountLockoutTimeSpan``).
+The following configures Account to be locked out for 10 minutes after 10 failed attempts.
 
-.. code-block:: c#
+.. literalinclude:: 2fa/sample/WebSMS/src/WebSMS/Startup.cs
+   :language: c#
+   :lines: 67-77
+   :emphasize-lines: 1-5
+   :dedent: 8
 
-    services.Configure<IdentityOptions>(options =>
-    {
-        options.Lockout.DefaultLockoutTimeSpan = TimeSpan.FromMinutes(10);
-        options.Lockout.MaxFailedAccessAttempts = 10;
-    });
+Debugging  Twilio
+-------------------
+If you're able to use the Twilio API, but you don't get an SMS message, try the following:
 
+1. Log in to the Twilio site and navigate to the **Logs** > **SMS & MMS Logs** page. You can verify that messages were sent and delivered.
+2. Use the following code in a console application to test Twilio:
 
-Next steps
```

Comment:
Why is this deleted? Users need to know how to set these secrets in Azure Web Apps